styles.css

```diff
@@ -0,0 +1,133 @@
+.img {
+  height: 20px;
+  width: 20px;
+  margin: 10px;
+}
+
+main {
+  max-width: 375px;
+  margin: auto;
+  height: fit-content;
+  background-image: url("Images/temp3.png");
+  background-repeat: no-repeat;
+  background-size: 100%;
+}
+
+.images {
+  justify-content: center;
+  text-align: center;
+  margin-top: 20px;
+  width: 100%;
+}
+
+h4 {
+  width: 90px;
+  height: 28px;
+  font-family: "Inter", sans-serif;
+  font-weight: 600;
+  font-size: 20px;
+  line-height: 28px;
+  display: inline-block;
+  align-items: center;
+  text-align: center;
+  color: #28352f;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  padding-left: 20px;
+}
+
+.header {
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  color: #172b4d;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  text-align: center;
+  margin-bottom: 0;
+  display: flex;
+  align-items: center;
+}
+
+.article {
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  text-align: center;
+  color: #3ec994;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  margin-top: 0;
+}
+
+.p {
+  width: 327px;
+  height: 164px;
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-size: 20px;
+  line-height: 28px;
+  text-align: center;
+  color: #172b4d;
+  flex: none;
+  order: 1;
+  flex-grow: 0;
+  justify-content: center;
+  margin: auto;
+}
+
+.space {
+  width: 100%;
+  height: 32px;
+}
+
+.more-space {
+  width: 100%;
+  height: 62px;
+}
+
+.top-bar {
+  max-width: 375px;
+  margin: auto;
+  overflow: hidden;
+  background-color: #fff;
+  top: 0;
+  width: 100%;
+  height: 92px;
+}
```

Comment:
- [Optional] Impressive How you align Items on the view. Also, it's part of the requirement to use the `flex-box` for the toolbar/header section to align items. It's good to manipulate float and other styling options, but you should also have the experience of sticking to the requirements. So, I strongly suggest you use 'flex-box' in your toolbar/heading.